server/views/pages/applications/comments/index.njk

```diff
@@ -63,7 +63,8 @@
               text: "Add a comment",
               classes: "govuk-label--s govuk-!-margin-bottom-4",
               isPageHeading: true
-            }
+            },
+            errorMessage: { text: errors.comment.text } if errors.comment.text else undefined
```

Comment:
Probably wants to match the earlier check `if errors.comment` without the `.text`

Reply:
Is it working in the other error pages? Personally I'd set a var and pass that as `errorMessage` rather than the conditional here.